.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,17 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
I intentionally checked this in. Inspection profiles, different from most `.idea` files, help teams stay on the same page as far as what static analysis errors are important to pay attention to.